PRCS/prc-1.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Paima Achievement Interface
+description: Interface for sharing in-game achievements.
+author: seba@paimastudios.com, edward@paimastudios.com
+status: open
+created: 2023-11-08
+---
+
+## Abstract
+
+A open cross-game standard achievement specification to gamify onchain participation. 
+This specification allows to interact through standard http methods.  
+
+## Motivation
+
+Most games have a achievement system, but there are not compatible: vendors and tools have adapt to each game. By implementing this standard, developer spend no or less time implementing the format itself, vendors can use the format for gamification apps, explorers, prizes, etc.
```

Comment:
we should add more to cover some key points like why our chosen standard is 
- easily indexable
- self-hostable
- accessible from the game itself (ex: areas in the game you can only reach if you've completed an achievement)

Reply:
```
This open-specification does not depend on any specific platform game host, technology, or language, and can be completely self-hosted. 

The achievement content is easily indexable for games and API consumers, in a recognizable format, allowing caching and generating useful tools for end users. 

This achievement system can be used by the target game itself for unlocking functionalities such as opening new areas, triggering game-progress, or giving away prizes. Third parties may also consume API to expand on its actions, which are compatible with on-chain games.
```